samples/error-correction/bit-flip-code/BitFlipCode.qs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.Quantum.Samples.BitFlipCode {
     // In this sample, we build on the discussion in the quantum error
     // correction section of the developers' guide:
 
-    //     https://docs.microsoft.com/quantum/libraries/error-correction
+    //     https://docs.microsoft.com/quantum/user-guide/libraries/standard/error-correction
```

Comment:
Thanks for catching this, looks like it got missed when we demoted the libraries in the documentation TOC.